godotproject/sfx/explosion_sfx.wav.import

```diff
@@ -0,0 +1,21 @@
+[remap]
```

Comment:
The .wav files themselves don't seem to exist in the branch. Can you add them in?

Reply:
yep, will do!